packages/terra-text/src/Text.jsx

```diff
@@ -0,0 +1,66 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import classNames from 'classnames';
+import 'terra-base/lib/baseStyles';
+import './Text.scss';
+
+const propTypes = {
+  /**
+   * Needs descriptions
+   */
+  children: PropTypes.node.isRequired,
+  /**
+   * The color of the text. Accepts a CSS color value.
```

Comment:
Nidpick:

CSS color value sounds rather specific. Perhaps it's better to say 'Accepts any color value parseable by CSS'?